pkg/utils/utils.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import "io/ioutil"
+
+func GetComponentNamespace() (string, error) {
+	nsBytes, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
+	if err != nil {
+		return "open-cluster-management", err
```

Comment:
you may don't need return err, just fall back to the default namespace

Reply:
and we already have this in the `helpers` https://github.com/open-cluster-management/cluster-proxy-addon/blob/main/pkg/helpers/helpers.go#L11, you can use it

Reply:
Ok, sure.